### DISCUSSION TITLE
[DOCS]: Regex for title validation used in `createDraftListing` isn't enough.

### DISCUSSION THREAD

**Discussion by SrdjanPaunovic** [etsy/open-api#709] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Link to Documentation Issue

https://developer.etsy.com/documentation/reference#operation/createDraftListing

### What would you like to change?

Can you provide a **valid regex** that we can use for title validation on our end? Provided regex in the documentation for `createDraftListing` is not covering everything.

### Why are you proposing this change?

I am getting errors while calling `createDraftListing` with an irregular title (that I don't know how to validate properly).

**Comment by etsyachristensen** [comment]
@SrdjanPaunovic We aren't able to debug this without some more info.

First, what is the title you are attempting to set?
Second, what error(s) are you getting when trying to create the listing?

**Comment by SrdjanPaunovic** [comment]
### 1.Example
Title: **Èxample titlé**
I'm getting this validation error message: _You cannot include any of the following characters: $^`._

### 2.Example
Tilte: AD AD AD AD 
I'm getting this validation error message: _Your title cannot contain more than 3 all capital words._

In documentation we only have this regex: **/[^\p{L}\p{Nd}\p{P}\p{Sm}\p{Zs}™©®]/u**  which is not covering every situation you validating on your site.

**Comment by etsyachristensen** [comment]
@SrdjanPaunovic We are still tracking this for a resolution. We are closing the issues board in favor of the discussions board, so I'll convert this now to a new discussion.

**Comment by DemiImp** [comment]
@etsyachristensen - Thank you for transferring my issue to a discussion topic.

Issue #622  also included changes for API endpoints that returned an array rather than a JSON object.  I also requested that the docs clearly state which API endpoints return an array rather than a javascript object.

I also suggested that instead of returning an array, it should return something similar to this:

`{"error":"unable to update listing","error_list":[{"path":"\/title\/1","type":"invalid_characters","message":"contains invalid characters"}]}`
